src/main/scala/io/opentargets/etl/backend/literature/spark/Helpers.scala

```diff
@@ -0,0 +1,142 @@
+package io.opentargets.etl.backend.literature.spark
+
+import com.typesafe.scalalogging.LazyLogging
+import io.opentargets.etl.backend.Configuration.{LiteratureModelConfiguration, OTConfig}
+import org.apache.spark.SparkConf
+import org.apache.spark.ml.feature.{Word2Vec, Word2VecModel}
+import org.apache.spark.ml.linalg.Vector
+import org.apache.spark.ml.linalg.Vectors.norm
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.types.{ArrayType, DataType, StructField, StructType}
+import org.apache.spark.sql.{Column, DataFrame, SparkSession}
+
+import scala.util.Random
+
+object Helpers extends LazyLogging {
```

Comment:
Let's rename this so we're not shadowing the Helpers object from another package. 

I think that after the following comments we're only going to be left with a couple of methods, which might be better placed elsewhere anyway, so we might be able to do away with this object completely.

Reply:
The class was removed after the other refactoring the class had only one method and it was moved to the backend/spark/Helpers.scala